openapi.yaml

```diff
@@ -24003,10 +23996,16 @@ components:
           x-linode-filterable: true
           type: string
           enum:
-          - pending
```

Comment:
We combine ticketing, ticketed, scheduled, and ready statuses into 'pending', so the end user will only see pending, started, and completed.